filebeat/scripts/module/docs.asciidoc

```diff
@@ -1,3 +1,30 @@
+:modulename: {module}
+
 == {module} module
 
 This is the {module} module.
+
+include::../include/what-happens.asciidoc[]
+
+[float]
+=== Compatibility
+
+TODO: document with what versions of the software is this tested
+
+
+include::../include/running-modules.asciidoc[]
+
+[float]
+=== Example dashboard
+
+This module comes with a sample dashboard. For example:
+
+TODO: include an image of a sample dashboard
+
+include::../include/configuring-intro.asciidoc[]
+
+TODO: provide an example configuration
+
```

Comment:
There's an example in config-option-intro.asciidoc that uses the `fileset_ex` attribute to resolve a fileset name:

````
TIP: When you specify a setting at the command line, remember to prefix the
setting with the module name, for example, +{modulename}.{fileset_ex}.var.paths+
instead of +{fileset_ex}.var.paths+.
````

So...I would add a TODO item before line 54.  Something like:

```
TODO: Set the fileset_ex attribute to the name of any fileset in your module. For example:
:fileset_ex: access
```